vm/src/obj/objint.rs

```diff
@@ -732,6 +748,17 @@ fn div_ints(vm: &VirtualMachine, i1: &BigInt, i2: &BigInt) -> PyResult {
     }
 }
 
+#[inline]
+fn divmod(vm: &VirtualMachine, i1: &BigInt, i2: &BigInt) -> PyResult<(BigInt, BigInt)> {
+    if i2.is_zero() {
+        Err(vm.new_zero_division_error("integer division by zero".to_string()))
+    } else {
+        let modulo = (i1 % i2 + i2) % i2;
```

Comment:
Did you have a look at this: https://docs.rs/num-bigint/0.2.2/num_bigint/struct.BigInt.html#method.div_rem ?

Reply:
div_rem was the original implementation which didn't match to python. I fixed them to div_floor and div_mod_floor, thanks.